orc8r/lib/go/service/service.go

```diff
@@ -106,7 +109,10 @@ func NewServiceWithOptionsImpl(moduleName string, serviceName string, serverOpti
 
 	// Check if service was started with print-grpc-payload flag or MAGMA_PRINT_GRPC_PAYLOAD env is set
 	if printGrpcPayload || util.IsTruthyEnv(PrintGrpcPayloadEnv) {
-		ls := logCodec{encoding.GetCodec(grpc_proto.Name)}
+		ls := logCodec{
+			protoCodec: encoding.GetCodec(grpc_proto.Name),
+			logMode:    strings.TrimSpace(fmt.Sprintf(os.Getenv(PrintGrpcPayloadEnv))),
```

Comment:
logMode should ideally be an enum field type or equivalent, and be given a default, or set based on `PrintGrpcPayloadEnv`

Reply:
Note that the idea for this was to maintain backwards compatibility 

Current code enables grpc printing as long as long as you have any value different than 0 or False. 

I can change that and make it an enum (so only 0, 1, 2... will be good for that). But then this may be stop working for anybody that is using it on a different way